src/chrome/render-asset.sh

```diff
@@ -8,7 +8,7 @@ OPTIPNG="$(command -v optipng)" || true
 if "$INKSCAPE" --help | grep -e "--export-png" > /dev/null; then
   EXPORT_FILE_OPTION="--export-png"
 else
-  EXPORT_FILE_OPTION="--export-file"
+  EXPORT_FILE_OPTION="--export-filename"
```

Comment:
do here the same kind of check as above, `if "$INKSCAPE" --help | grep -e "--export-filename" > /dev/null; then`